httpcore/_async/connection_pool.py

```diff
@@ -291,11 +292,19 @@ async def aclose(self) -> None:
         Close any connections in the pool.
         """
         async with self._pool_lock:
+            requests_still_in_flight = bool(self._requests)
+
             for connection in self._pool:
                 await connection.aclose()
             self._pool = []
             self._requests = []
 
+            if requests_still_in_flight:
+                raise RuntimeError(
+                    "The connection pool was closed while some HTTP "
```

Comment:
Instead of "some", it should be possible to get a number correct (`len(self._requests)`)? I know I have seen this in other exceptions or repr. I certainly appreciated that when I have seen it.